src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -185,27 +183,6 @@ protected override bool Receive(object message)
         {
             switch (message)
             {
-                case KafkaConsumerActorMetadata.Internal.Assign assign:
```

Comment:
These are really `ISubscriptionRequest` messages so they've been moved to the `HandleSubscriptions` method.